pytamaro/point.py

```diff
@@ -0,0 +1,103 @@
+"""
+`Point` type and functions to represent a point int he 2 dimensional space.
+"""
+
+from dataclasses import dataclass
+from typing import Tuple
+
+
+@dataclass
+class Point:
+    """
+    Represents a point in 2-dimensional space.
+    """
+    x: float    # pylint: disable=invalid-name
+    y: float    # pylint: disable=invalid-name
+
+    def __init__(self, x: float, y: float):  # pylint: disable=invalid-name
```

Comment:
No need to have an explicit `__init__` with dataclasses ([it gets automatically generated](https://docs.python.org/3/library/dataclasses.html)).